roottest/root/io/evolution/equivalent/ioReadRuleEquivalentCode.h

```diff
@@ -4,7 +4,7 @@ class A:public TObject{
 
 public:
    A(){}
-   ClassDef(A,1)
+   ClassDefOverride(A,1)
```

Comment:
Same here - as long there are no new virtual methods override, there is no sense to use `ClassDefOverride`

Reply:
It derives from TObject, which has itself a ClassDef with virtual functions.

within the derived class, if you define ClassDef again, it redefines those virtual.
So you need ClassDefOverride.